pkg/cni/plugin/plugin.go

```diff
@@ -135,37 +133,47 @@ func checkKmesh(client kubernetes.Interface, pod *v1.Pod) (bool, error) {
 	return false, nil
 }
 
-func kmeshCtlByClassid(client kubernetes.Interface, pod *v1.Pod) error {
-	classIDPathPrefix := "/sys/fs/cgroup/net_cls/kubepods"
-
-	qosClass := strings.ToLower(string(pod.Status.QOSClass))
-	podUID := "pod" + pod.UID
-	classidFile := "net_cls.classid"
-	netClsPath := path.Join(classIDPathPrefix, string(qosClass), string(podUID), classidFile)
-
-	file, err := os.OpenFile(netClsPath, os.O_RDWR|os.O_APPEND, 0)
-	if err != nil {
-		err = fmt.Errorf("failed to open net cls path: %v, %v", netClsPath, err)
-		log.Error(err)
-		return err
+func kmeshCtlByNetCookie(ns string) error {
+	execFunc := func(netns.NetNS) error {
+		/*
+		 * Attempt to connect to a special IP address. The
+		 * connection triggers the cgroup/connect4 ebpf
+		 * program and records the netns cookie information
+		 * of the current connection. The cookie can be used
+		 * to determine whether the netns is managed by Kmesh.
+		 * 0.0.0.1:929(0x3a1) is "cipher key" for cgroup/connect4
+		 * ebpf program.
+		 */
+		_, _ = net.Dial("tcp", "0.0.0.1:929")
```

Comment:
Is "0.0.0.1:929" a constant value? If so, you can use a `const` variable to avoid hard-coding here.